eosiojavarpcprovider/build.gradle

```diff
@@ -1,140 +1,67 @@
-apply plugin: 'com.android.library'
-//Remove 'com.jfrog.artifactory' plugin if you are not using Artifactory
-apply plugin: "com.jfrog.artifactory"
-apply plugin: 'maven-publish'
+plugins {
+    id 'java-library'
+    //Remove 'com.jfrog.artifactory' plugin if you are not using Artifactory
+    id 'com.jfrog.artifactory' version '4.10.0'
+    id 'maven-publish'
 
-android {
-    compileSdkVersion 28
-
-    defaultConfig {
-        minSdkVersion 21
-        targetSdkVersion 28
-        versionCode 1
-
-        testInstrumentationRunner "android.support.test.runner.AndroidJUnitRunner"
-
-    }
-
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
-        }
-    }
-
-    lintOptions {
-        abortOnError false
-    }
 }
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     api 'one.block:eosiojava:0.1.2'
 
     implementation 'org.jetbrains:annotations:17.0.0'
+    compile 'com.googlecode.json-simple:json-simple:1.1.1'
```

Comment:
implementation now supersedes compile
"Why no compile configuration?
The Java Library Plugin has historically used the compile configuration for dependencies that are required to both compile and run a project’s production code. It is now deprecated, and will issue warnings when used, because it doesn’t distinguish between dependencies that impact the public API of a Java library project and those that don’t. "